.gitignore

```diff
@@ -3,10 +3,6 @@
 public/Lychee-front/node_modules/
 public/Lychee-front/package-lock.json
 
-public/Lychee-front/node_modules/
-public/Lychee-front/bower_components/
-public/Lychee-front/package-lock.json
-
```

Comment:
What happens if we stop ignoring those files?  More over this makes me wonder if we should simply ignore the whole directory `Lychee-front`. It is a GIT submodule at last.  However, I was too lazy to look into the official GIT docs how `.gitignore` of the parent project and submodules interact with each other.

Reply:
2 of the 3 are ignored just above, so only whatever that is. I haven't seen it.